packages/control-property-editor/src/panels/outline/Tree.tsx

```diff
@@ -294,8 +294,12 @@ export const Tree = (): ReactElement => {
             <></>
         );
         const isExtensionPoint = item?.controlType === 'sap.ui.extensionpoint';
+        const hasDefaultContent = item?.hasDefaultContent || false;
+
         const tooltipId = `tooltip--${item?.name}`;
 
+        const cellName = hasDefaultContent ? `${item?.name} (has default content)` : item?.name;
+
```

Comment:
Shouldn't we use texts from i18n?

Reply:
Added in: [c86f608](https://github.com/SAP/open-ux-tools/pull/1867/commits/c86f608325dd2d9f9b520beffc14d1a6122242c5)